addons/mail/tests/test_mail_template.py

```diff
@@ -153,7 +153,7 @@ def test_mail_template_acl_translation(self):
         with self.assertRaises(AccessError):
             employee_template.with_context(lang='fr_FR').subject = '{{ object.foo }}'
 
-        employee_template.with_context(lang='fr_FR').sudo().subject = '{{ object.foo }}'
+        employee_template.with_context(lang='fr_FR').sudo().subject = '{{ \'foo\' }}'
```

Comment:
Mmm, can you explain the change?

Reply:
This change was made since the "object.foo" issue occurred and there is no "foo" field in the "res.partner" model, so I passed a static string instead.

Reply:
Looks like this test made no sense ... maybe check object.name or something similar ?

Reply:
done